core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-configuration/components/dot-experiments-configuration-goals/dot-experiments-configuration-goals.component.spec.ts

```diff
@@ -138,14 +145,20 @@ describe('DotExperimentsConfigurationGoalsComponent', () => {
 
     it('should show a confirmation to delete a goal', () => {
         spyOn(store, 'deleteGoal');
-        const vmMock$: { experimentId: string; goals: Goals; status: StepStatus } = {
+        const vmMock$: {
+            experimentId: string;
+            goals: Goals;
+            status: StepStatus;
+            isExperimentADraft: boolean;
+        } = {
             experimentId: EXPERIMENT_ID,
             goals: GoalsMock,
             status: {
                 status: ComponentStatus.IDLE,
                 isOpen: false,
                 experimentStep: null
-            }
+            },
+            isExperimentADraft: true
```

Comment:
Any test of check if the tooltip is loaded?

Reply:
done